tests/test_tng.py

```diff
@@ -121,88 +123,109 @@ def test_tng_example_getitem_bool_TypeError(cls, TNG_EXAMPLE):
             for ts in tng[slx]:
                 ts.step
 
-@pytest.mark.skip(reason="FAILING")
+
 def test_tng_example_natoms(TNG_EXAMPLE_DATA, TNG_EXAMPLE):
     with pytng.TNGFileIterator(TNG_EXAMPLE) as tng:
         assert TNG_EXAMPLE_DATA.natoms == tng.n_atoms
 
+
+def test_README_example(TNG_EXAMPLE):
+    with pytng.TNGFileIterator(TNG_EXAMPLE, 'r') as tng:
+        positions = np.ndarray((tng.n_atoms, 3), dtype=np.float32)
+        for ts in tng:
+            time = ts.get_time()
+            positions = ts.get_positions(positions)
+
+
+def test_DOCS_example(TNG_EXAMPLE):
+    with pytng.TNGFileIterator(TNG_EXAMPLE, 'r') as tng:
+        positions = tng.make_ndarray_for_block_from_name("TNG_TRAJ_POSITIONS")
+        for ts in tng[0:len(tng):tng.block_strides["TNG_TRAJ_POSITIONS"]]:
+            positions = ts.get_positions(positions)
+
+
 def test_tng_example_tng_example_first_positions(
     TNG_EXAMPLE_DATA, TNG_EXAMPLE
 ):
     with pytng.TNGFileIterator(TNG_EXAMPLE) as tng:
-        print(tng.block_strides)
-        first_frame = tng[0].pos
-        print(first_frame)
-        print(TNG_EXAMPLE_DATA.first_frame)
-        assert np.array_equal(TNG_EXAMPLE_DATA.first_frame, first_frame)
+        pos = np.zeros((15, 3), dtype=np.float32)
```

Comment:
ok, so it looks like you can use the high-level NumPy array instantiators just fine then